tests/schema/__app/schema.prisma

```diff
@@ -7,16 +7,24 @@ model Bubble {
   id        String   @id @default(uuid())
   createdAt DateTime @default(now())
   members   User[]
+  private   Boolean
 }
 
 model User {
   id        String @id @default(cuid())
   posts     Post[]
   firstName String
   lastName  String
+  location  Location
   // likes     Post[]
 }
 
+model Location {
+  id        Int @id @default(autoincrement())
+  country   String
+  city      String
+}
+
```

Comment:
I made some changes to the integration test to exercise areas of schema generation that I wasn't sure about before.

1. boolean fields and how those show up for e.g. filtering
2. singular (non-list) relations